docs/docs/getting-started/example.md

```diff
@@ -3,6 +3,21 @@ id: example
 title: Getting Started
 ---
 
+## Installtion
+
+Install with npm
+
+```sh
+  npm i next-auth
+```
+
+Install with yarn
+
+```sh
+  yarn add next-auth
+```
```

Comment:
Also we have a plugin which will automatically make an `npm`/`yarn` tabbed instruction thing.

For example:

```
You can upgrade to the new version by running:

` ` ` bash npm2yarn
npm install next-auth
` ` `
```

Will generate:

![image](https://user-images.githubusercontent.com/7415984/169073945-258b1504-a33c-4488-9c5d-2bbc5e0ccd1c.png)
![image](https://user-images.githubusercontent.com/7415984/169073974-c31083c6-fe47-4cff-9cfb-d329593fa2e8.png)


See `/docs/getting-started/upgrade-to-v4.md:16`